src/components/WordGroupSelector/WordGroupIcon.js

```diff
@@ -0,0 +1,13 @@
+import React from "react";
```

Comment:
Also can you move each component with their css into their own folder so we're following along w/ what Jason did